_stbt/core.py

```diff
@@ -875,7 +875,7 @@ def match_text(self, text, frame=None, region=Region.ALL,
         xml, region = _tesseract(frame, region, mode, lang, _config,
                                  user_words=text.split())
         if xml == '':
-            return TextMatchResult(ts, False, None, frame, text)
+            result = TextMatchResult(ts, False, None, frame, text)
```

Comment:
This isn't *just* a refactoring -- it is also changing the behaviour. Now it will proceed to run the code below, and `lxml.etree.fromstring('')` will raise an XMLSyntaxError. (This is caught by one of our unit tests -- if you run `make check` you should see it fail.)